src/voterData/voterFile/util/voterFile.util.ts

```diff
@@ -0,0 +1,341 @@
+import { Logger } from '@nestjs/common'
+import {
+  CustomFilter,
+  CustomVoterFile,
+  VoterFileType,
+} from '../voterFile.types'
+import { CampaignWith } from 'src/campaigns/campaigns.types'
+
+const logger = new Logger('Voter File Utils')
+
+export function typeToQuery(
+  type: VoterFileType,
+  campaign: CampaignWith<'pathToVictory'>,
+  customFilters?: Pick<CustomVoterFile, 'channel' | 'filters' | 'purpose'>,
+  justCount?: boolean,
+  fixColumns?: boolean,
+) {
+  const state = campaign.details.state
+  let whereClause = ''
+  let nestedWhereClause = ''
+  let l2ColumnName = campaign.pathToVictory?.data.electionType
+  const l2ColumnValue = campaign.pathToVictory?.data.electionLocation
+
+  if (l2ColumnName && l2ColumnValue) {
+    // value is like "IN##CLARK##CLARK CNTY COMM DIST 1" we need just CLARK CNTY COMM DIST 1
+    let cleanValue = extractLocation(l2ColumnValue, fixColumns)
+    if (fixColumns) {
+      console.log('before fix columns:', l2ColumnName)
+      l2ColumnName = fixCityCountyColumns(l2ColumnName)
+      console.log('after fix columns:', l2ColumnName)
```

Comment:
either pass in a `Logger` to log these, or just delete them.

Looks like debug logging, so I'd have preference toward the latter.